### PR TITLE
Typescript removed, Dashboard UI implemented

### PR DESCRIPTION
**Pull request by jappreetS**
- Removed Typescript and related dependencies
- Dashboard UI implementation

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,14 +6,10 @@
...
     "@testing-library/user-event": "^12.1.10",
-    "@types/jest": "^26.0.15",
-    "@types/node": "^12.0.0",
-    "@types/react": "^17.0.0",
-    "@types/react-dom": "^17.0.0",
+    "node-sass": "^5.0.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
-    "typescript": "^4.1.2",
```
Comment (varora1406) [inline package.json:16]: we can still keep typescript part. maybe Dhariya and I can write typescript files for some places.
I have managed typescript + js combo, so it should be fine.

I don't want to degrade from js to typescript, if I can manage some part at least with typescript😁 . It's super useful

File: src/components/JumboButton/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <button
+      className="JumboButton linear-gradient d-flex align-items-center"
+      type="button"
+    >
+      <img alt={altText} src={iconSrc} />
+      <div className="text-align-left">
+        <div className="JumboButton-primaryText">{primaryText}</div>
+        <div className="JumboButton-secondaryText">{secondaryText}</div>
+      </div>
```
Comment (varora1406) [inline src/components/JumboButton/index.js:15]: ideally looking for a UI system, out there, would reduce burden of maintaining some codebase from our side
+ 
we would get some cool UI features with them by defined.

But leave final call on agreement with @AlwaysHungrie  and @jappreetS

**Review by varora1406 — CHANGES_REQUESTED** [review]

File: src/components/JumboButton/index.js (hunk 2 above)
Comment (AlwaysHungrie) [reply src/components/JumboButton/index.js:15]: https://material-ui.com/

Comment (jappreetS) [reply src/components/JumboButton/index.js:15]: will change this, for now merging as it is

Comment (jappreetS) [reply src/components/JumboButton/index.js:15]: let the designs get finalized.

**Pull request merged**